third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -44,9 +45,8 @@ resource "google_composer_environment" "test" {
 
 ### With GKE and Compute Resource Dependencies
 
-**NOTE** To use service accounts, you need to give `role/composer.worker` to the service account on any resources that may be created for the environment
-(i.e. at a project level). This will probably require an explicit dependency
-on the IAM policy binding (see `google_project_iam_member` below).
+**NOTE** To use custom service accounts, you need to give at least `role/composer.worker` to the service account being used Composer Nodes on the Composer project.
+You may need to assign additional roles depending on what the Airflow DAGs will be running.
```

Comment:
I'm not familiar with GKE resources, so I would just be confused by "Airflow DAGs" and "Composer Nodes" compared to the old docs. Is there a different way to phrase this? Or is it something where if I were doing this, I would absolutely know what these phrases refer to?

Reply:
For example, if you write a DAG that talks to Bigquery, you need to assign additional roles. roles/composer.worker doesn't include BQ roles.

https://cloud.google.com/composer/docs/how-to/using/writing-dags#gcp_operators

I clarified it a little bit.